pages/linux/flameshot.md

```diff
@@ -0,0 +1,25 @@
+# flameshot 
+
+> Screenshot utility with a gui interface.
+> Supports basic image editing, such as; text,shapes,colors,imgur.
+> More information: <hhttps://flameshot.js.org/>.
+
+- Launch flameshot in gui mode:
+
+`flameshot launcher`
+
+- Click and drag screenshot:
+
+`flameshot gui`
+
+- Full screen screenshot:
+
+`flameshot full`
+
+- Set save path for flameshot:
+
+`flameshot full --path ~/Pictures`
```

Comment:
We can use the `{{token_syntax}}` here to indicate that the user can change the value .
```suggestion
`flameshot full --path {{path/to/directory}}`
```

Reply:
>We can use the {{token_syntax}} here to indicate that the user can change the value .

Thank you I didn't know what that meant. Awesome. Very helpful :smiley: